web/client/components/TOC/fragments/settings/ModelTransformation.jsx

```diff
@@ -0,0 +1,136 @@
+
+/*
+ * Copyright 2024, GeoSolutions Sas.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormGroup, ControlLabel, InputGroup } from 'react-bootstrap';
+import DebouncedFormControl from '../../../misc/DebouncedFormControl';
+import Message from '../../../I18N/Message';
+
+/**
+ * ModelTransformation. This component shows the model transformation options available
+ * @prop {object} layer the layer options
+ * @prop {object} onChange callback on every on change event
+ */
+function ModelTransformation({
+    layer,
+    onChange
+}) {
+    const changeCenterModelHandler = (value) => {
+        const feature = layer?.features?.[0];
+        const [longitude, latitude, height] = feature?.geometry?.coordinates || [0, 0, 0];
+        const updatedCenter = {
+            longitude,
+            latitude,
+            height,
+            ...value
+        };
+        const newBbox = {
+            ...layer?.bbox,
+            bounds: {
+                minx: updatedCenter.longitude - 0.001,
+                miny: updatedCenter.latitude - 0.001,
+                maxx: updatedCenter.longitude + 0.001,
+                maxy: updatedCenter.latitude + 0.001
+            }
+        };
+        onChange('features', [
+            {
+                properties: {},
+                ...feature,
+                type: 'Feature',
+                geometry: {
+                    type: 'Point',
+                    coordinates: [
+                        updatedCenter.longitude,
+                        updatedCenter.latitude,
+                        updatedCenter.height
+                    ]
+                }
+            }
+        ]);
+        onChange('bbox', newBbox);
+    };
+    if (layer?.type !== 'model') {
+        return null;
+    }
+    const feature = layer?.features?.[0];
+    const [longitude, latitude, height] = feature?.geometry?.coordinates || [0, 0, 0];
+    return (
+        <div style={{ margin: '0 -8px' }}>
+            <FormGroup className="form-group-flex">
+                <ControlLabel><Message msgId="layerProperties.modelLayer.modelCenterLng"/></ControlLabel>
+                <InputGroup style={{ maxWidth: 210 }}>
+                    <DebouncedFormControl
+                        type="number"
+                        name={"modelCenterLng"}
+                        value={longitude}
+                        fallbackValue={0}
+                        onChange={(val)=> {
+                            changeCenterModelHandler({
+                                longitude: val !== undefined
+                                    ? parseFloat(val) : 0
+                            });
+                        }}
+                    />
+                    <InputGroup.Addon>DD</InputGroup.Addon>
```

Comment:
I'm not sure if DD is clear. This means decimal degrees, right?
Probably it's better to remove the unit of measure in this case.
Also in the search bar we are not adding the unit of measure

![image](https://github.com/geosolutions-it/MapStore2/assets/19175505/342ba4cb-f588-444c-bdae-427f2d22100f)